ext/standard/image.c

```diff
@@ -1148,6 +1149,123 @@ static struct gfxinfo *php_handle_webp(php_stream * stream)
 }
 /* }}} */
 
+/* {{{ php_handle_avif
+ * There's no simple way to get this information - so, for now, this is unsupported.
+ * Simply return 0 for everything.
+ */
+static struct gfxinfo *php_handle_avif(php_stream * stream) {
+		struct gfxinfo * result;
```

Comment:
Indentation seems to be wrong; should be 1 tab per level in C files. Please also check elsewhere.

Reply:
Nice catch. I fixed the double indentation here, and I cleaned it up in one another spot. I don't see this anywhere else, but please let me know if you do...

Reply:
heh, another review found one more spot...